src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -1,12 +1,13 @@
 package edu.kis.vh.nursery;
 
 import edu.kis.vh.nursery.factory.DefaultRhymersFactory;
-import edu.kis.vh.nursery.list.IntegerLinkedList;
+import edu.kis.vh.nursery.stack.IntArrayStack;
+import edu.kis.vh.nursery.stack.IntegerLinkedList;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class RhymersJUnitTest {
-	static final int EMPTY_STACK_VALUE = -1;
+	static final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).